### ISSUE TITLE
Experiment #004: Correlated data and small samples - take 3

### ISSUE THREAD

**Issue by AyronK** [AyronK/softmax-dnn-example#11]
Repeated #10 but with multiple instances of same deck.

## Assumptions
1. Full deck has 3 card slots
2. There is 10 total cards in the pool.
3. Each deck is taken 3 times into data set:
```
                // Class 1 decks
                {1, 1, 0, 0, 1, 0, 0, 0, 0, 0},  * 3
                {1, 1, 0, 0, 0, 1, 0, 0, 0, 0},  * 3
                {1, 1, 0, 0, 0, 0, 1, 0, 0, 0},  * 3
                {1, 1, 0, 0, 0, 0, 0, 1, 0, 0},  * 3
                // Class 2 decks  
                {0, 0, 1, 1, 0, 0, 1, 0, 0, 0},  * 3
                {0, 0, 1, 1, 0, 0, 0, 1, 0, 0},  * 3
                {0, 0, 1, 1, 0, 0, 0, 0, 1, 0},  * 3
                {0, 0, 1, 1, 0, 0, 0, 0, 0, 1},  * 3
```
Same rules for correlation.

## Process
1. For each deck remove one "card" and take two as input and one as output
    - input:   [1,0,0,0,1,0,0,0,0,0] 
    - output: [0,0,1,0,0,0,0,0,0,0] 
2. Shuffle the data of 72 total (_8 decks * 3 cards * 3 takes_) options
3. Take 18 options as learning set and 54 as testing set **It is now 1:3. In previous experiment it was 3:1, so we have more real ratio of learning set in comparison to testing one.**
4. Teach network
5. Test accuracy

**Comment by AyronK** [comment]
## Result
Below the comparison between avarage of 100 tries accuracy for two different networks

### Activation Network
- 5 hidden neurons
- LearningRate = 0.07,
- Momentum = 0.3

**Learing set accuracy:** 82,33%
**Testing set accuracy:** 71,06%

### Deep Belief Neural Network
- 2 hidden layers, 5 neurons each
- Unsupervised
   * LearningRate = 0.07,
   * Momentum = 0.3
- Supervised
   * LearningRate = 0.07,
   * Momentum = 0.2

**Learing set accuracy:** 82,080%
**Testing set accuracy:** 66,69%

**Comment by AyronK** [comment]
## Conclusion

- The result are slightly better.
